### PR TITLE
DOC: update README for 2023

### PR DESCRIPTION
**Pull request by deniederhut**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,25 +106,24 @@ which ensures relevance to the community that created them.
 
 The most effective way to contact the Proceedings Co-Chairs for issues related to this GitHub repository is to use GitHub's issues and "@"-mentioning the Co-Chairs.
 
-In 2022, the Proceedings Co-Chairs are
+In 2023, the Proceedings Co-Chairs are
 - Meghann Agarwal (@mepa)
 - Chris Calloway (@cbcunc)
 - Dillon Niederhut (@deniederhut)
-- David Shupe (@stargaser)
```
Comment (mepa) [inline README.md:113]: Is this list complete?

File: README.md [hunk 2]
```diff
@@ -247,22 +246,22 @@ upstream	https://github.com/scipy-conference/scipy_proceedings.git (fetch)
...
+#### Getting the latest branch
 
 - Fetch the latest version of the `scipy_proceedings` repo
     - `git fetch upstream`
-- Check out the upstream `2022` branch
-    - `git checkout -b 2022 --track upstream/2022`
+- Check out the upstream `2023` branch
+    - `git checkout -b 2023 --track upstream/2023`
 
-#### Creating a new branch based off of `2020`
```
Comment (mepa) [inline README.md:257]: Good idea to remove the year from headings!

**Review by mepa — COMMENTED** [review]
Hey Dillon, thanks for updating the README! Could you please also check lines 13 and 16 for references to 2022?

**Comment by deniederhut** [comment]
Note to self -- missed some stuff at the top of the file:

```
**All** communication between authors and reviewers should be civil and
respectful. There are no exceptions to this rule. Please see the
[SciPy2022 Code of Conduct](https://www.scipy2022.scipy.org/code-of-conduct)
for more info.
```

**Review by mepa — APPROVED** [review]

**Pull request merged**